src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -884,13 +883,23 @@ private static bool ProcessPeformanceCounterEnabled()
             }
             catch (Exception)
             {
-                // Ignore exceptions, and just assume the counter is enabled.
+                // Ignore exceptions. This may occur if the user doesn't have admin privileges.
+                // The computer may be running with a guest user with process performance counters
+                // disabled. If so, try connecting to a remote machine. If the error contains any
+                // indication that process performance counters are disabled, then we will know.
+                try
+                {
+                    Process.GetProcesses(Guid.NewGuid().ToString("N"));
+                }
+                catch (InvalidOperationException ex)
+                {
+                    return !ex.Message.Contains("Process performance counter is disabled");
```

Comment:
This test will fail on machines where .NET is localized. .NEt core is not, but .NET Framework is.